Automattic/sensei
# Fixes for Take Course button

Closes #3736

### Changes proposed in this Pull Request

* Removes disabled styles from style preview buttons.
* Changes the label of the button.
* Adds a preview for button blocks and a description to 'Contact Teacher' block.
* There is another issue with the style style selector. Any style changes are applied to these components too. I'll look into that as part of another PR.

### Testing instructions

* Create a new take course button.
* Check that the disabled styles are not applied to the styles selector.
* Check that the button is created with a left alignt by default.
* Check that the label on the alignment toolbar is correct.
* Check the preview and the description of button blocks.
